packages/codex-harness/scripts/codex_review.py

```
@@ -0,0 +1,244 @@
+        *trust_flags,
+        REVIEW_PROMPT,
+    ]
+    events_path = run_dir / "events.jsonl"
+    progress_path = run_dir / "progress.log"
+    with (
+        open(input_diff_path, encoding="utf-8") as diff_f,
+        open(events_path, "w", encoding="utf-8") as events_f,
+        open(progress_path, "w", encoding="utf-8") as progress_f,
+    ):
+        try:
+            completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)